html/semantics/scripting-1/the-script-element/microtasks/evaluation-order-1-nothrow-static-import.any.js

```diff
@@ -1,4 +1,4 @@
-// META: global=dedicatedworker-module,sharedworker-module
+// META: global=dedicatedworker-module,sharedworker-module,shadowrealm-in-window,shadowrealm-in-shadowrealm,shadowrealm-in-dedicatedworker,shadowrealm-in-sharedworker
```

Comment:
With the improved error reporting in #49387 I realized these didn't run in `shadowrealm-in-serviceworker` and `shadowrealm-in-audioworklet` because the fake dynamic import isn't actually module code :confounded: I'm not sure how important that is though.